go.mod

```diff
@@ -73,3 +71,6 @@ replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 
 // Override reference causing proxy error.  Otherwise it attempts to download https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190813000000-74dc4d7220e7.info
 replace golang.org/x/net => golang.org/x/net v0.0.0-20190923162816-aa69164e4478
+
+// Override cortex to get ARM support.
+replace github.com/cortexproject/cortex => github.com/cyriltovena/cortex v0.0.0-20191114231131-3e991d87dfa9
```

Comment:
This is now fixed on Cortex master.

Reply:
I know I was busy on something else Owen took over